samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/ShadowContainerSamplePage.WinUI.xaml

```diff
@@ -310,7 +310,51 @@
 							<StackPanel Spacing="8">
 								<TextBlock Text="Complex Shadow Shapes" Style="{StaticResource TitleTextBlockStyle}" />
 
-								<StackPanel>
+										<!--
+									Should be the same as: https://developer.mozilla.org/fr/docs/Web/CSS/CSS_backgrounds_and_borders/Box-shadow_generator
+									element {
+									width: 300px;	height: 100px;	background-color: #7A67F8;	position: relative;
+									box-shadow:
+									6px -5px 5px 0px #E88BAB,
+									inset 5px -5px 5px 0px #94DB6D,
+									inset -20px -10px 10px 10px #000000;
+									}
+								-->
+								<utu:ShadowContainer Margin="0,30" Background="{StaticResource UnoColor}">
+									<utu:ShadowContainer.Shadows>
+										<utu:ShadowCollection>
+											<utu:Shadow BlurRadius="10"
```

Comment:
just for ease of reading, can we order the properties the same as the css?
[inset,]x,y,blur,spread,color,[opacity]
and keep \#000000 as is